webapps/api/oauth2/tokenview.py

```diff
@@ -47,7 +52,12 @@ def post(self, request, *args, **kwargs):
         """
         Handles token post
         """
-        logger = Logger('api')
+        expiration = Configuration.get('ovs/framework/api/oauth|expiration', default={})
+        if not isinstance(expiration, dict):
+            self.logger.exception('The oauth expiration configuration is not an object. Falling back to defaults')
+            expiration = {}
+        expiration_password = expiration.get('password', self.EXPIRATION_PASSWORD)
```

Comment:
Why a dict in a dict?
now the value must look like
{'expiration': {'password': 3600, 'client': 3600}}

Why not just use this structure:
{'password_expiration': 3600, 'client_expiration': 3600}

And while writing this i was thinking whether this is a security risk?  Every user is able to modify the timeouts of tokens?  Do we want this?

Reply:
Does ovs config get/set work for any user except for root and ovs?
And with that logic, a user can wreck havoc in other parts of our code too